hasura/migrations/1588758007277_init/up.sql

```diff
@@ -1,4 +1,3 @@
-CREATE EXTENSION citext;
```

Comment:
you need to remove it from the down.sql

Reply:
but the create extension is needed to work on vanilla postgres

Reply:
mmm ok. the issue is that atm on azure PG only superuser can create extensions, that's why we add it on database creation

Reply:
i try some tricks